### PR TITLE
Integrated scope fix for scout rifle.

### PR DESCRIPTION
**Pull request by Rivet-the-Zombie**
Rather than using the wonky irremovable mod system, we'll just give
scope functionality to the base weapon and remove the separate scope
item.

### CODE REVIEW

**Comment by Coolthulhu** [comment]
Default mods aren't irremovable. The scope that spawns with the rifle is more useful than the rifle itself, as it can be taken off and attached to something the player has ammo for.

File: CataclysmWin.cbp [hunk 1]
```diff
@@ -328,11 +328,6 @@
 			<Add library="gdi32" />
 			<Add library="winmm" />
 		</Linker>
-		<ExtraCommands>
```
Comment (mugling) [inline CataclysmWin.cbp]: ??

Comment (Rivet-the-Zombie) [inline CataclysmWin.cbp]: Screwed up; will fix.

**Comment by Rivet-the-Zombie** [comment]
>The scope that spawns with the rifle is more useful than the rifle itself, as it can be taken off and attached to something the player has ammo for.

It wasn't meant to be removable in the first place, see:

>integrated suppressor and digital scope

**Pull request merged**